service/src/main/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelper.java

```diff
@@ -26,12 +27,16 @@ public static String formatJudgeTitleAndName(JudgeAndLegalAdvisor judgeAndLegalA
     }
 
     private static String mapJudgeOrAdvisor(JudgeAndLegalAdvisor judgeAndLegalAdvisor) {
-        if (isBlank(judgeAndLegalAdvisor.getJudgeLastName()) && isBlank(judgeAndLegalAdvisor.getJudgeFullName())) {
+        if (isBlank(judgeAndLegalAdvisor.getJudgeLastName())
+            && judgeAndLegalAdvisor.getJudgeTitle() != MAGISTRATES) {
             return "";
         }
 
         if (judgeAndLegalAdvisor.getJudgeTitle() == MAGISTRATES) {
```

Comment:
Would it be better to use a switch statement in this situation?

Reply:
yeah could use switch on .getJudgeTitle() for nicer code, but not too concerned as this story is fairly pressed for time + needs to be tested still. I'll let adam decide if he wants to refactor this part